ArtOfIllusion/src/artofillusion/texture/ProceduralTexture3D.java

```diff
@@ -235,6 +185,14 @@ public Texture duplicate()
     tex.antialiasing = antialiasing;
     return tex;
   }
+  
+  /** Get the list of parameters for this texture. */
+
+  @Override
+  public TextureParameter[] getParameters()
```

Comment:
You've moved this method from above `duplicate()` to below. Any particular reason?

Reply:
I do this to made this and ProceduralTexture2D methods order the same to be easily compared. Same for allowParameters() and allowViewAngle()...

Reply:
Don't make changes like that. This is sort of like reformatting - it does not belong in a PR that is focused on something else.